vendor: bump etcd/raft to pick up memory usage fix

tbg opened the pull request:
Pick up coreos/etcd#9887, which avoids pulling
the uncommitted portion of the Raft log into memory.

Release note (bug fix): Alleviate a scenario in which a large number of
uncommitted Raft commands could cause memory pressure at startup time.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/27009)
<!-- Reviewable:end -->

tschottdorf commented:
@bdarnell I looked through the changes that we picked up here and they didn't seem worrisome, but still - how proactive do we want to be about putting this on release-2.0?

tschottdorf commented:
TFTR. Commit message and PR title fixed.

bors r=bdarnell

tbg committed:
vendor: bump etcd/raft to pick up memory usage fix

Pick up coreos/etcd#9887, which avoids pulling the uncommitted portion
of the Raft log into memory.

Release note (bug fix): Alleviate a scenario in which a large number of
uncommitted Raft commands could cause memory pressure at startup time.

tschottdorf commented:
Rebased.
bors r=bdarnell

bdarnell commented:
So we put this in 2.0.4 but never merged it to master? Oops.

tschottdorf commented:
Yep.

On Thu, Jul 19, 2018 at 3:31 PM Ben Darnell <notifications@github.com>
wrote:

> So we put this in 2.0.4 but never merged it to master? Oops.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/27009#issuecomment-406277076>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE135JwNa258kDs3eSiz2jlSVtdrgjACks5uIIpLgaJpZM4U5eu8>
> .
>
-- 

-- Tobias
